packages/mdc-textfield/_variables.scss

```diff
@@ -14,15 +14,14 @@
 // limitations under the License.
 //
 
-$mdc-text-field-error-on-light: #d50000;
-$mdc-text-field-divider-on-light: rgba(black, .12);
-$mdc-text-field-disabled-border-on-light: rgba(#231f20, .26);
-$mdc-text-field-disabled-icon-on-light: rgba(black, .3);
-$mdc-text-field-underline-on-light: rgba(black, 1);
-$mdc-text-field-underline-on-light-idle: rgba(black, .5);
-$mdc-text-field-light-background: rgba(white, 1);
-$mdc-text-field-light-label: rgba(black, .6);
-$mdc-text-field-light-placeholder: rgba(black, .38);
+$mdc-text-field-error: #d50000;
+$mdc-text-field-divider: rgba(black, .12);
+$mdc-text-field-disabled-border: rgba(#231f20, .26);
+$mdc-text-field-disabled-icon: rgba(black, .3);
+$mdc-text-field-underline: rgba(black, 1);
```

Comment:
Can we rename this to `$mdc-text-field-underline-hover`, since that's what it's used for?